src/ring_theory/polynomial/gauss_lemma.lean

```diff
@@ -25,15 +28,18 @@ Gauss's Lemma is one of a few results pertaining to irreducibility of primitive
 
 open_locale non_zero_divisors polynomial
 
-variables {R : Type*} [comm_ring R] [is_domain R]
+variables {R : Type*} [comm_ring R]
 
 namespace polynomial
 section normalized_gcd_monoid
-variable [normalized_gcd_monoid R]
 
+lemma is_primitive_of_dvd' {p q : R[X]} (hp : is_primitive p) (hq : q ∣ p) : is_primitive q :=
```

Comment:
What's the difference with [polynomial.is_primitive.is_primitive_of_dvd](https://leanprover-community.github.io/mathlib_docs/ring_theory/polynomial/content.html#polynomial.is_primitive.is_primitive_of_dvd)? This looks more general.